### DISCUSSION TITLE
Automatic updates/refresh from Nordigen

### DISCUSSION THREAD

**Discussion by anthurta** [firefly-iii/firefly-iii#8067] [Firefly III]
Hello,

I have everthing set up and the first import is correctly done via Nordigen. My installation is on HomeAssistant as an addon (although I don't think I will leave there, but for testing purposes it was the easiest way).

So, I was reading the documentation and I have a very simple (and probably stupid) question: using the Data Importer, how can I refresh the data coming from Nordigen? I see the data importer is refreshing but only looking for files in some directories and I am not sure if I can really refresh through Nordigen or that is just the one-time-first import.

Thanks in advance.

**Comment by JC5** [comment]
https://docs.firefly-iii.org/data-importer/advanced/automation/
